jupyterlab_gcpscheduler/src/components/jobs_panel_widget.tsx

```diff
@@ -137,59 +154,55 @@ export class GcpScheduledJobsPanel extends React.Component<Props, State> {
   componentDidUpdate(prevProps: Props) {
     if (
       this.props.isVisible &&
-      !(prevProps.isVisible || this.state.isLoading)
+      !(prevProps.isVisible || this.state.runsTab.isLoading)
     ) {
-      this._getRunsAndSchedules(this.state.rowsPerPage);
+      this._getRuns(this.state.rowsPerPage);
+      this.setState({ tab: 0, page: 0 });
```

Comment:
Could we leave the user on the tab they were previously on but set the page to zero? That way, if they created a new Schedule, they would see it appear if they moved from the file browser to the Executor panel.

Reply:
Ah okay. Can fix that